src/Microsoft.AspNet.Mvc.Core/Logging/FilterActionInvokerLoggerExtensions.cs

```diff
@@ -0,0 +1,56 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.AspNet.Mvc.Logging
+{
+    public static class FilterActionInvokerLoggerExtensions
+    {
+        private static Action<ILogger, string, Exception> _authorizationFailure;
+        private static Action<ILogger, string, Exception> _resourceFilterShortCircuit;
+        private static Action<ILogger, string, Exception> _actionFilterShortCircuit;
+        private static Action<ILogger, string, Exception> _exceptionFilterShortCircuit;
+
+        static FilterActionInvokerLoggerExtensions()
+        {
+            _authorizationFailure = LoggerMessage.Define<string>(
+                LogLevel.Warning,
+                1,
+                "Authorization failed for the request at filter '{AuthorizationFilter}'.");
+            _resourceFilterShortCircuit = LoggerMessage.Define<string>(
+                LogLevel.Verbose,
+                2,
+                "Request was short circuited at resource filter '{ResourceFilter}'.");
+            _actionFilterShortCircuit = LoggerMessage.Define<string>(
+                LogLevel.Verbose,
+                3,
+                "Request was short circuited at action filter '{ActionFilter}'.");
+            _exceptionFilterShortCircuit = LoggerMessage.Define<string>(
+                LogLevel.Verbose,
+                4,
+                "Request was short circuited at exception filter '{ExceptionFilter}'.");
+        }
+
+        public static void AuthorizationFailure(this ILogger logger, string filterName)
```

Comment:
We should probably just pass in the filter to logging as `object`. The logging system will ToString it, which gives the existing behavior of printing the full name.